template.yaml

```diff
@@ -0,0 +1,368 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
The IaC code should go in the cambian-services-v2 repository.  We were thinking it should go in a new folder under Core
`Core\Organization\ECS\CDR\HAPI-FHIR`